doc/src/site/sphinx/migration_guide.rst

```diff
@@ -0,0 +1,72 @@
+Migration Guide
+===============
+
+Migration guide between major Sparkling Water versions
+
+From 3.26 To 3.28
+-----------------
+
+String instead of enums in Sparkling Water Algo API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- In scala, Setters on the pipeline wrappers for H2O algorithms now accepts strings in places where they accepted
+  enum values before. Before, we called, for example:
+
+.. code-block:: scala
+
+    import hex.genmodel.utils.DistributionFamily
+    val gbm = H2OGBM()
+    gbm.setDistribution(DistributionFamily.multinomial)
+
+
+Now, the correct code is:
+
+.. code-block:: scala
+
+    val gbm = H2OGBM()
+    gbm.setDistribution("multinomial")
+
+which makes the Python and Scala consistent. Both upper case and lower case values are valid and if a wrong
+input is entered, user is warned with correct possible values.
+
+Switch to Java 1.8 on Spark 2.1
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sparkling Water for Spark 2.1 now requires Java 1.8 and higher.
+
+DRF exposed into Sparkling Water Algorithm API
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+DRF is now exposed in the Sparkling Water. Please see our documentation to learn how to use it :ref:`drf`.
+
+Single value in prediction column
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default name of the prediction column is changed to ``prediction`` from ``prediction_output``. Also, the
+prediction output now contains always directly the predicted value. In case of regression issue the predicted numeric value
```

Comment:
`Also, the
prediction output now contains always directly the predicted value. In case of regression issue the predicted numeric value and in case of classification the predicted label.` Can you rephrase it?

Reply:
Rephrased, please check if it make more sense now